### ISSUE TITLE
[Bug] Merge Into Case search by title not working

### ISSUE THREAD

**Issue by mphbig** [TheHive-Project/TheHive#1983]
### Request Type
Bug

### Work Environment

| Question                   | Awnser                                |
| -------------------------- | ------------------------------------- |
| OS version (server)        | Debian 9   |
| OS version (client)        | Windows 10            |
| Virtualized  Env.          | True                          |
| Dedicated RAM              | 16 GB                                 |
| vCPU                       | 8                       |
| TheHive version / git hash | 4.1.2               |
| Package Type               | DEB |
| Database                   | Cassandra               |
| Index type                 | Lucene                |
| Attachments storage        | Local                  |
| Browser type & version     | Firefox and Chrome latest                         |

### Problem Description
When trying to merge an alert into a case using the "Merge Into Case" button, if you search your case by case title, you will find that it shows no case at all. Even after a reindex.

The request "/api/v1/query?name=get-case-for-merge" seems to never return anything.

### Steps to Reproduce
1. Open a new alert
2. Try to merge it using the "Merge Into Case" button
3. Search for a case using its title

### Complementary information
TheHive logs, shows nothing out of the ordinary, it reponds 200 as if everything went well.

**Comment by mphbig** [comment]
Workarround identified : using * (wildcard) character allows to find cases

**Comment by torsolaso** [comment]
You must use minus always an between  wilcards. For me, * string * is working (no spaces)

**Comment by aim4r** [comment]
> 
> 
> You must use lowercase always an between wilcards. For me, * string * is working (no spaces)

That works for some cases, but when searching for an IP (say for example `*10.10.10.10*`) it does not work.

**Comment by mphbig** [comment]
Little heads up on the workarround, I learned the hard way that using wildcards tends to bypass the indexes, so TheHive was behaving like pre-4.1 and was painfully slow.

It prints out this log when using wildcards :
```
[WARN] from org.janusgraph.graphdb.transaction.StandardJanusGraphTx in application-akka.actor.default-dispatcher-12 [0000012c|08ca3135] Query requires iterating over all vertices [()]. For better performance, use indexes
```

**Comment by nadouani** [comment]
> Little heads up on the workarround, I learned the hard way that using wildcards tends to bypass the indexes, so TheHive was behaving like pre-4.1 and was painfully slow.
> 
> It prints out this log when using wildcards :
> 
> ```
> [WARN] from org.janusgraph.graphdb.transaction.StandardJanusGraphTx in application-akka.actor.default-dispatcher-12 [0000012c|08ca3135] Query requires iterating over all vertices [()]. For better performance, use indexes
> ```

Is this valid in 4.1.4?

**Comment by nadouani** [comment]
This looks fixed in 4.1.4

**Comment by mphbig** [comment]
We are on 4.1.4 and still got the issue :(

Could it be because we got very old data (+ 1 year old) and not everything is indexed ?
Or am I mistaking and everything is indexed ?
